subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/idea/IdeaJavaLanguageSettingsIntegrationTest.groovy

```diff
@@ -420,8 +420,8 @@ idea {
     void "language levels specified in properties files are ignored"() {
         given:
         file('gradle.properties') << """
-sourceCompatibility=1.3
```

Comment:
Perhaps it may make sense to leave the non-java prefixed properties here for this test as well as include the new ones?

Reply:
Good catch!
Done